src/main/java/bio/terra/model/StudyJsonConversion.java

```diff
@@ -0,0 +1,208 @@
+                .name(study.getName())
+                .description(study.getDescription())
+                .createdDate(study.getCreatedDate().toString());
+    }
+
+    public static StudyModel studyModelFromStudy(Study study) {
+        return new StudyModel()
+                .id(study.getId().toString())
+                .name(study.getName())
+                .description(study.getDescription())
+                .createdDate(study.getCreatedDate().toString())
+                .schema(studySpecificationModelFromStudy(study));
+    }
+
+    public static StudySpecificationModel studySpecificationModelFromStudy(Study study) {
+        return new StudySpecificationModel()
+                .tables(study.getTables()
+                        .stream()
+                        .map(table -> tableModelFromStudyTable(table))
+                        .collect(Collectors.toList()))
+                .relationships(study.getRelationships()
+                        .stream()
+                        .map(rel -> relationshipModelFromStudyRelationship(rel))
+                        .collect(Collectors.toList()))
+                .assets(study.getAssetSpecifications()
+                        .stream()
+                        .map(spec -> assetModelFromAssetSpecification(spec))
+                        .collect(Collectors.toList()));
+    }
+
+    public static StudyTable tableModelToStudyTable(TableModel tableModel) {
+        StudyTable studyTable = new StudyTable()
+                .setName(tableModel.getName());
+        studyTable
+                .setColumns(tableModel.getColumns()
+                        .stream()
+                        .map(columnModel -> columnModelToStudyColumn(columnModel).setInTable(studyTable))
+                        .collect(Collectors.toList()));
+        return studyTable;
+    }
+
+    public static TableModel tableModelFromStudyTable(StudyTable studyTable) {
+        return new TableModel()
+                .name(studyTable.getName())
+                .columns(studyTable.getColumns()
+                        .stream()
+                        .map(column -> columnModelFromStudyColumn(column))
+                        .collect(Collectors.toList()));
+    }
+
+    public static StudyTableColumn columnModelToStudyColumn(ColumnModel columnModel) {
+        return new StudyTableColumn()
+                .setName(columnModel.getName())
+                .setType(columnModel.getDatatype());
+    }
+
+    public static ColumnModel columnModelFromStudyColumn(StudyTableColumn tableColumn) {
+        return new ColumnModel()
+                .name(tableColumn.getName())
+                .datatype(tableColumn.getType());
+    }
+
+    public static StudyRelationship relationshipModelToStudyRelationship(
+            RelationshipModel relationshipModel,
+            Map<String, StudyTable> tables) {
+        return new StudyRelationship()
+                .setName(relationshipModel.getName())
```

Comment:
This all looks great! One thing that I think we should address outside of this PR: this fluent pattern is different than the one our `Model` classes are using. Any method prefixed with `set` in those is a void setter, instead the "fluent" version of the setter is just the property name itself as a function. 

See the difference between the function above and this one. I think it will make things more readable to keep things consistent.

Reply:
ah - i was using intellij generate... setter dialog and choosing the builder style. it prepends the set keyword. if we want this to be consistent and be able to use intellij generate, we either need to modify the intellij template or the swagger generate.